dask/tests/test_order.py

```diff
@@ -39,6 +97,7 @@ def test_ordering_keeps_groups_together(abcde):
     assert abs(o[(a, 1)] - o[(a, 3)]) == 1
 
 
+@pytest.mark.skip()
```

Comment:
Why skip this test?

Reply:
I'll delete this test.

```
    b0    b1  b2
    |      \  /
    a0      a1
```

This topology is a little ambiguous. Earlier versions asserted that we start with `a0` first but from a parallel computing POV it would actually be beneficial to start with `a1` while it has zero impact for serial computing. The only thing we'd want to avoid it to start with either branch and switch to the other one before the branch is done.

I think this test case is too simple to be meaningful